driver/tests/swe_roe/mms_dx1.yaml

```diff
@@ -0,0 +1,62 @@
+physics:
```

Comment:
This is a good start. We discussed perhaps making another YAML file format for the MMS driver that would allow us greater flexibility in configuring the problem of interest, right? Is that still interesting?

Reply:
As discussed during the call, the additional flexibility will be added in a future PR.